elixir/apps/web/lib/web/live/resources/components.ex

```diff
@@ -6,7 +6,7 @@ defmodule Web.Resources.Components do
       if Domain.Accounts.traffic_filters_enabled?(account) do
         attrs
       else
-        Map.put(attrs, "filters", %{"all" => %{"enabled" => "true", "protocol" => "all"}})
+        Map.put(attrs, "filters", %{})
```

Comment:
I think we can remove all this custom attrs mapping code now.

Reply:
Will check now

Reply:
Yeah, we still need them with our current schema because the embedded schema and form fields don't quite align.

I'll open an issue, maybe we can refactor at later point to migrate the embedded schema to a format more suited to represent the form it backs. That will require Gateway changes though, so probably not worth overhauling right now.